### PR TITLE
Add nf-test to minimap2 index

### LINKED ISSUE #5163: [FEATURE] Add nf-test to minimap2/index
### Is your feature request related to a problem? Please describe

Add nf-test to minimap2/index.

### Describe the solution you'd like

_No response_

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by fellen31**
## PR checklist

Closes #5163  

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [X] `nf-core modules test <MODULE> --profile docker`
    - [X] `nf-core modules test <MODULE> --profile singularity`
    - [X] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by maxulysse — COMMENTED** [review]
Can you remove the old pytest tests?

File: modules/nf-core/minimap2/index/tests/tags.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minimap2/index:
+  - "modules/nf-core/minimap2/index/**"
```
Comment (LouisLeNezet) [inline modules/nf-core/minimap2/index/tests/tags.yml:2]: Not sure that the " are necessary here.

**Review by LouisLeNezet — COMMENTED** [review]
Was pytest available for this module ?
If so you need to delete them in tests/modules/nf-core/minimap2/index
And also in tests/conf/pytest.yml :)
Otherwise good to go for me ;)

**Comment by fellen31** [comment]
Thanks @maxulysse @LouisLeNezet! Didn't realize about the pytests, hopefully fixed now!

**Comment by LouisLeNezet** [comment]
Nicely done !

**Pull request merged**